src/workerd/io/actor-cache.c++

```diff
@@ -2794,6 +2815,8 @@ kj::Promise<void> ActorCache::flushImplUsingTxn(
   // avoid the data changing out from under us while we wait.
   co_await waitForPastReads();
 
+  auto writeObserver = recordStorageWrite(hooks, clock);
```

Comment:
The is the only time that the `recordStorage*` function isn't measuring the same time as a `util::DurationExceededLogger`.  (The corresponding logger starts recording on line 2920.)  Should they be coincident?  If not, can you comment why they're not the same?

Reply:
No important reason. I was in some sense future-proofing against any awaits being added between `waitForPastReads` and when we wait on our response, and in some sense attempting to account for the time spent sending the storage operations as opposed to just waiting on their responses. But that time should be negligible, so I'll just stick with consistency I guess.